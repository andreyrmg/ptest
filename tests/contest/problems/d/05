1
603
111 2 31898