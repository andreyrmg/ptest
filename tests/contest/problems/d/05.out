1 281 7
